<?= $this->extend('common/layout') ?>
<?= $this->section('content') ?>



<br>
<h2 class="wiki-top-header">
    Information
</h2>
<br>

    Below is the full text of the letter to the editor regarding GRASSIUS. ( Original article is located 
    <A HREF="http://www.plantphysiol.org/cgi/doi/10.1104/pp.108.128504">here
    </A>.)


<br>
<br>
<h2 class="wiki-section-header">
    A Recommendation for Naming Transcription Factor Proteins in the Grasses
</h2>  
<br>
    <STRONG>
      </NOBR>
    <NOBR>John Gray
    </NOBR>, 
    <NOBR>Michael Bevan
    </NOBR>, 
    <NOBR>Thomas Brutnell
    </NOBR>, 
    <NOBR>C. Robin Buell
    </NOBR>, 
    <NOBR>Karen Cone
    </NOBR>, 
    <NOBR>Sarah Hake
    </NOBR>, 
    <NOBR>David Jackson
    </NOBR>, 
    <NOBR>Elizabeth Kellogg
    </NOBR>, 
    <NOBR>Carolyn Lawrence
    </NOBR>, 
    <NOBR>Susan McCouch
    </NOBR>, 
    <NOBR>Todd Mockler
    </NOBR>, 
    <NOBR>Stephen Moose
    </NOBR>, 
    <NOBR>Andrew Paterson
    </NOBR>, 
    <NOBR>Thomas Peterson
    </NOBR>, 
    <NOBR>Daniel Rokshar
    </NOBR>, 
    <NOBR>Glaucia Mendes Souza
    </NOBR>, 
    <NOBR>Nathan Springer
    </NOBR>, 
    <NOBR>Nils Stein
    </NOBR>, 
    <NOBR>Marja Timmermans
    </NOBR>, 
    <NOBR>Guo-Liang Wang
    </NOBR> and 
    <NOBR>Erich Grotewold 
      <SUP>*
      </SUP>
    </NOBR>
    </STRONG>
  <br>
  <br>
  Department of Biological Sciences, University of Toledo, Toledo, Ohio 43606 (J.G.)
  <br> Department of Cell and Molecular Biology, John Innes Center, Norwich Research Park, Norwich NR4 7UH, United Kingdom (M.B.)
  <br> Boyce Thompson Institute, Cornell University, Ithaca, New York 14853 (T.B.)
  <br>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824 (C.R.B.)
  <br>Division of Biological Sciences (K.C.) and Department of Biology (E.K.), University of Missouri, Columbia, Missouri 65211
  <br> Plant Gene Expression Center, Albany, California 94710 (S.H.)
  <br>Cold Spring Harbor Laboratory, Cold Spring Harbor,New York 11724 (D.J., M.T.)
  <br> Corn Insects and Crop Genetics Research Unit, Ames, Iowa 50201 (C.L.)
  <br>Cornell University, Ithaca, New York 14853 (S.M.)
  <br>Department of Botany and Plant Pathology and Center for Genome Research and Biocomputing,Oregon State University, Corvallis, Oregon 97331 (T.M.)
  <br> Department of Crop Sciences and Energy Biosciences Institute, University of Illinois, Urbana-Champaign, Illinois 61801 (S.M.)
  <br> Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia 30602 (A.P.)
  <br> Department of Genetics, Development and Cell Biology, Iowa State University, Ames, Iowa 50011&#8211 3260 (T.P.)
  <br>Department of Energy Joint Genome Institute, Walnut Creek, California 94598 (D.R.)
  <br> Instituto de Qu&iacute;mica, Departamento de Bioqu&iacute;mica, Universidade de S&atilde;o Paulo, S&atilde;o Paulo, Brazil (G.M.S.)
  <br> Center for Plant and Microbial Genomics, Department of Plant Biology, University of Minnesota, Saint Paul, Minnesota 55108 (N. Springer)
  <br> Leibniz-Institute of Plant Genetics and Crop Plant Research, Genebank Department, 06466 Gatersleben, Germany (N. Stein)
  <br> and Department of Plant Pathology (G.-L.W.) and Department of Plant Cellular and Molecular Biology and Plant Biotechnology Center (E.G.), The Ohio State University, Columbus, Ohio 43210
  <br>
  <br>* Corresponding author
  <br>
<h2 class="wiki-section-header"></h2>  
  <br>
  <div style="text-indent: 30pt;" align=justify>
    Transcription factors are central for the exquisite temporal and spatial expression patterns of many genes. These proteins are characterized by their ability to be tethered to particular regulatory sequences in the genes that they control. While many other proteins participate in the regulation of gene expression, we limit our definition of transcription factors here to proteins that often contain a characteristic structural motif, the DNA-binding domain, which is involved in recognizing a short (usually 4&#8211;8bp) DNA sequence. Based on the structure of the DNA-binding domain, transcription factors are classified into 50 to 60 different families, and in plants, 5% to 7% of all the protein-encoding genes are transcription factors, making them, collectively, perhaps the largest functional class of proteins.
  </div>
  <div style="text-indent: 30pt;" align=justify>
    The availability of protein sequence information from several plant genomes that have been fully or partially sequenced over the past few years is creating an urgent need to develop a set of criteria for naming and identifying members of large protein families. A common nomenclature would facilitate better communication among scientists, working not just on a particular plant system,but also across different plant species. This need is becoming particularly acute in the grasses, where some members have a rich genetic history (e.g. maize [
    <I>Zea mays
    </I>], rice [
    <I>Oryza sativa
    </I>],barley [
    <I>Hordeum vulgare
    </I>]) and genes have been named for more than 100 years according to the phenotypes of the corresponding mutations, often leading to multiple names for the same gene. In other grasses that do not have such a genetic heritage, such as sugarcane (
    <I>Saccharum
    </I> ssp.), genes are characterized primarily by EST accession numbers, often with multiple nonoverlapping sequences corresponding to one gene. Gene nomenclature is governed by species-specific communities in many cases (e.g. maize, rice,sorghum [
    <I>Sorghum bicolor
    </I>]), but such committees do not exist for all species with significant sequence data. In general, names of proteins follow the names of their corresponding genes.
  </div>
  <div style="text-indent: 30pt;" align=justify>
    In Arabidopsis (
    <I>Arabidopsis thaliana
    </I>), shortly after the completion of the genome sequence, criteria were developed to provide unique names to all transcription factors, often in a family-by-family strategy (e.g. Stracke et al., 2001
    <A HREF="#BIB14">
      <IMG BORDER=1 WIDTH=8 HEIGHT=7 ALT="Go"
           SRC="images/old/fig-down.gif">
        </A>; Jakoby et al., 2002
      <A HREF="#BIB10">
        <IMG BORDER=1 WIDTH=8 HEIGHT=7 ALT="Go"
             SRC="images/old/fig-down.gif">
          </A>; Baileyet al., 2003
        <A HREF="#BIB1">
          <IMG BORDER=1 WIDTH=8 HEIGHT=7 ALT="Go"
               SRC="images/old/fig-down.gif">
            </A>). These identifiers are of the form AtXXXyyy, where <i>At</i> provides the species identifier At = Arabidopsis, important when describing transcription factors from multiple plants; <i>XXX</i> corresponds to a two to five or more letter code for the particular transcription factor family; and <i>yyy</i> corresponds to an arbitrary number between one and the total number of members in that particular family. These nomenclature conventions were rapidly embraced by the community, facilitating communication in publications and allowing the development of databases that compile information on Arabidopsis regulatory proteins (Davuluriet al., 2003
          <A HREF="#BIB4">
            <IMG BORDER=1 WIDTH=8 HEIGHT=7 ALT="Go"
                 SRC="images/old/fig-down.gif">
              </A>; Guo et al., 2005
            <A HREF="#BIB9">
              <IMG BORDER=1 WIDTH=8 HEIGHT=7 ALT="Go"
                   SRC="images/old/fig-down.gif">
                </A>).
              </div>
            <div style="text-indent: 30pt;" align=justify>
              Here, we propose adoption of similar synonyms for proteins corresponding to transcription factors across the grasses, with a goal of having a uniform naming system as already developed for <i>Arabidopsis</i>. Such names are not meant to replace the often-familiar names for many proteins, but rather to provide synonyms that can link information about all members of the gene family. Briefly, each transcription factor will be identified by a two-letter codecorresponding to the species (e.g. Zm for 
              <I>Z. mays
              </I>, Sb for 
              <I>S.bicolor
              </I>, and Os for 
              <I>O. sativa
              </I>), followed by the transcription factor family name and by a number that represents its position within the family. The two-letter species code should suffice for now to unequivocally describe the grasses for which transcription factors have been identified, but clearly in the future it will need to be expanded to three or four letters, as the number of species being included in studies increases. Alternatively, the two-letter code reflecting the species name could be used for organisms whose genomes are currently under study, and if needed, a new two-letter abbreviation, which is not necessarily consistent with the genus and specific epithet will be madeup for new ones. Following this criterion, sugarcane will be identified by the Sc letter code, as agreed by the respective community.
            </div>
            <div style="text-indent: 30pt;" align=justify>
              Numbers will be assigned arbitrarily, and whenever possible,the numbers should provide a historic perspective of the order in which transcription factors have been first identified. For example, since maize KN1 and C1 correspond to the founding members of their respective families (HD and MYB, respectively), they are assigned the number 1. When a transcription factor has already been numbered, every possible effort should be made to consider that number as part of the new name, e.g. maize Zm38 (Frankenet al., 1994
              <A HREF="#BIB6">
                <IMG BORDER=1 WIDTH=8 HEIGHT=7 ALT="Go"
                     SRC="images/old/fig-down.gif">
                  </A>) should become ZmMYB38. Since it is realized that many transcription factors are known by their genetic names, this nomenclature will permit the use of various different synonyms. For example, KNOTTED1, which would be ZmHD1, where HD corresponds to the homeodomain family, could also be identified as ZmHD1(KN1) when in need to highlight the genetic locus, 
                <I>KN1
                </I>, for which the protein is also often known. Similarly C1, ZmMYB1, could be also identified as ZmMYB1(C1). Where the same name has already been used for different family members, then an alternative name(s) will be assigned to avoid further confusion in the literature. For example, ZmMYB8 (Fornale et al., 2006
                <A HREF="#BIB5">
                  <IMG BORDER=1 WIDTH=8 HEIGHT=7 ALT="Go"
                       SRC="images/old/fig-down.gif">
                    </A>), which is supported by a complete cDNA will remain ZmMYB8, whereas the partial ESTs MYB8 (Jiang et al., 2004
                  <A HREF="#BIB11">
                    <IMG BORDER=1 WIDTH=8 HEIGHT=7 ALT="Go"
                         SRC="images/old/fig-down.gif">
                      </A>) and ZmMYB-IP20 (Rabinowicz et al.,1999
                    <A HREF="#BIB13">
                      <IMG BORDER=1 WIDTH=8 HEIGHT=7 ALT="Go"
                           SRC="images/old/fig-down.gif">
                        </A>), which both correspond to the same protein, will be assigneda new name (e.g. ZmMYB67).
                      </div>
                    <div style="text-indent: 30pt;" align=justify>
                      While it would be attractive for this new naming system to provide information about orthologous pairs across species, as for example ZmMYB32 being the closest relative to OsMYB32, this was considered impractical and perhaps misleading for several reasons. First, the genomes are not yet completely sequenced or fully annotated, hence new transcription factors are likely to be identified in the future, significantly affecting the reconstruction of protein phylogenies. Second, different tree-building methods are likely to yield slightly different results, which would create significant confusion. Third, while it has been tempting to assume that high similarity is likely to correspond to the control of similar cellular processes, this has often not been the case, particularly when considering regulators of metabolic pathways (Grotewold, 2008
                      <A HREF="#BIB7">
                        <IMG BORDER=1 WIDTH=8 HEIGHT=7 ALT="Go"
                             SRC="images/old/fig-down.gif">
                          </A>).
                        </div>
                      <div style="text-indent: 30pt;" align=justify>
                        We propose that GRASSIUS (www.grassius.org) will serve as an initial centralized clearinghouse for transcription factor synonyms for the grasses, starting with maize, rice, sorghum, and sugarcane, following the criteria outlined above. GRASSIUS will provide a source of cross-reference between the new names, synonyms, National Center for Biotechnology Information accession codes for ESTs, and cDNAs and unique gene identifiers, as they become available. This will be achieved dynamically, ensuring that immediately after a new synonym has been given to a transcription factor, it will be reflected in GRASSIUS. The community is invited to comment on these assignments for a defined period of time (e.g. until the end of 2009), at which time the names will become official. The community will be presented with these guidelines and will be provided with ample opportunity to become aware and discuss these recommendations at conferences and meetings for the corresponding organisms. In addition, we will work with the various model organism and clade-oriented databases (e.g.MaizeGDB, BrachyBase, and Gramene) to ensure that the proper nomenclature is represented in these community resources as well. These databases will also serve as optimal clearinghouses for the respective organisms if GRASSIUS ceases to serve this purpose. Conflicts and issues that may arise with respect to how to name a particular transcription factor (or family oftranscription factors) will be opened for discussion to experts in the field. For example, if there is a disagreement on how to name a new family of transcription factors, scientists working with those specific transcription factors will be invited to comment.
                      </div> 
                      <br>
                      <A NAME="SEC1">
                        <!-- null -->
                      </A>
                        
                        

<br>
<h2 class="wiki-section-header">
    SOME PARTICULAR CASES
</h2>  
<br>
                        
                      <STRONG>Multiple Proteins from One Gene
                      </STRONG>
                      <br>
                      <div style="text-indent: 30pt;" align=justify>
                        Multiple transcripts derived from one gene are frequently present in plants, with more than 21% of the rice and Arabidopsis genes being alternatively spliced (Wang and Brendel, 2006
                        <A HREF="#BIB15">
                          <IMG BORDER=1 WIDTH=8 HEIGHT=7 ALT="Go"
                               SRC="images/old/fig-down.gif">
                            </A>). Several examples of transcription factors displaying alternate splice variants have also been described in maize (e.g. Grotewold etal., 1991
                          <A HREF="#BIB8">
                            <IMG BORDER=1 WIDTH=8 HEIGHT=7 ALT="Go"
                                 SRC="images/old/fig-down.gif">
                              </A>; Burr et al., 1996
                            <A HREF="#BIB2">
                              <IMG BORDER=1 WIDTH=8 HEIGHT=7 ALT="Go"
                                   SRC="images/old/fig-down.gif">
                                </A>). We recommend that transcription factor proteins derived from alternate spliced mRNAs be named with the .1, .2, .3 suffixes after the number of the protein. For example, the new synonyms corresponding to the two proteins derived from the alternatively spliced variants of maize 
                              <I>PERICARPCOLOR1
                              </I> (
                              <I>P1
                              </I>) would be ZmMYB3.1 and ZmMYB3.2. In those instances, as well as in cases when multiple gene models exist for a particular transcription factor gene, the suffixes in the protein will match those in the gene models. For instance, if the rice LOC_Os02g36880 gene shows four different gene models, from .1 to .4, then OsNAC1.1 should match with LOC_Os02g36880.1 and OsNAC1.4 should matchLOC_Os02g36880.4.
                              </div>
                            <BR>
                              <STRONG>Allelic Variants
                              </STRONG>
                              <br>
                              <div style="text-indent: 30pt;" align=justify>
                                The sequencing of multiple inbred lines/subspecies displaying significant natural variation makes it necessary to incorporate an option to represent from which allele a particular transcription factor protein sequence is derived. We propose that whenever necessary, a superscript is added. This superscript could represent the source of the allele, when known. For example, the P1 protein obtained from the W22 maize inbred could be represented as ZmMYB3.1
                                <SUP>W22
                                </SUP>, and that from B73 as ZmMYB3.1
                                <SUP>B73
                                </SUP>. When formatting issues prevent the use of the superscripts, then it would also be acceptable to use ^B73 to represent the allele. In that case, ZmMYB3.1
                                <SUP>B73
                                </SUP>and ZmMYB3.1^B73 would be equivalent. Such criterion could also be used to indicate, whenever known, whether a transcription factor protein sequence in rice is derived from the sequenced
                                <I>japonica
                                </I> genome (cv Nipponbare) or the sequenced 
                                <I>indica
                                </I> genome (cv 9311). Thus, the OsNAC6 factor, involved in biotic and abioticstress response in rice (Ohnishi et al., 2005
                                <A HREF="#BIB12">
                                  <IMG BORDER=1 WIDTH=8 HEIGHT=7 ALT="Go"
                                       SRC="images/old/fig-down.gif">
                                    </A>), could be OsNAC6
                                  <SUP>Nipp
                                  </SUP>(or OsNAC6
                                  <SUP>9311
                                  </SUP>) when intending to capture aspects of the protein that relate to variation. If the origin is not precisely known or a name/accession_ID is too cumbersome to be represented as a superscript, then numbers could be used to distinguish alleles (e.g. OsNAC6
                                  <SUP>1
                                  </SUP>, OsNAC6
                                  <SUP>2
                                  </SUP>, etc.), with cross-references to inbred/accession names maintained within species databases. Of course, within a species, a single transcription factor name (e.g. OsNAC6) will correspond to the products of corresponding gene models.
                                  </div>
                                <BR>
                                  <STRONG>Products from Tandem Gene Arrays
                                  </STRONG>
                                  <br>
                                  <div style="text-indent: 30pt;" align=justify>
                                    In some instances, for example the various alleles of the maize
                                    <I>p1
                                    </I> gene (Chopra et al., 1998
                                    <A HREF="#BIB3">
                                      <IMG BORDER=1 WIDTH=8 HEIGHT=7 ALT="Go"
                                           SRC="images/old/fig-down.gif">
                                        </A>), very similar (but not necessarily identical) proteins are encoded by individual members of a multigene array. In those instances, we recommend using letters (a, b, c) to indicate the proteins that come from each copy. For example, if three different copies of the 
                                      <I>p1
                                      </I> gene from B73 were shown to encode slightly different proteins, then those products would be identified as ZmMYB3
                                      <SUP>B73
                                      </SUP>a, ZmMYB3
                                      <SUP>B73
                                      </SUP>b, and ZmMYB3
                                      <SUP>B73
                                      </SUP>c.
                                      </div>
                                    <br>
                                    <A NAME="SEC2">
                                      <!-- null -->
                                    </A>          

<br>
<h2 class="wiki-section-header">
    GENE AND PROTEIN NOMENCLATURE
</h2>  
<br>
                                    <div style="text-indent: 30pt;" align=justify>
                                      The guidelines described here for naming transcription factors are expected to apply solely to proteins (or predicted open reading frames) and not necessarily to genes. Indeed, as the sequencing of various genomes progresses, nomenclature committees have been established to address the issue of how to name genes and gene products. It is therefore of paramount importance that the guidelines described here are in line with those being developed by the corresponding committees. Toward this objective, the criteria described here have already been discussed and accepted for the maize transcription factors by the Maize Genetics Nomenclature Committee (
                                      <A HREF="http://www.maizegdb.org/maize_nomenclature.php">http://www.maizegdb.org/maize_nomenclature.php
                                      </A>), by the Sugarcane Nomenclature Committee, and by the International Brachypodium Initiative (
                                      <A HREF="http://www.brachypodium.org/">http://www.brachypodium.org/
                                      </A>). The corresponding nomenclature committees will make these guidelines available to the respective communities.
                                    </div>
                                    <br>
                                    <A NAME="BIBL">
                                      <!-- null -->
                                    </A>
<br>
<h2 class="wiki-section-header">
    LITERATURE CITED
</h2>  
<br>
                                    <A NAME="BIB1">
                                      <!-- null -->
                                    </A>
                                    <B>Bailey PC, Martin C, Toledo-Ortiz G, Quail PH, Huq E, Heim MA, Jakoby M, Werber M, Weisshaar B
                                    </B> (2003) Update on the basic helix-loop-helix transcription factor gene family in 
                                    <I>Arabidopsis thaliana
                                    </I>. Plant Cell 
                                    <B>15:
                                    </B> 2497&#8211;2502
                                    <!-- HIGHWIRE ID="149:1:4:1" -->
                                    <br>
                                    <A NAME="BIB2">
                                      <!-- null -->
                                    </A>
                                    <B>Burr FA, Burr B, Scheffler BE, Blewitt M, Wienand U, Matz EC
                                    </B> (1996) The maize repressor-like gene 
                                    <I>intensifier1
                                    </I> shares homology with the 
                                    <I>r1
                                    </I>/
                                    <I>b1
                                    </I> multigene family of transcription factors and exhibits missplicing. Plant Cell 
                                    <B>8:
                                    </B> 1249&#8211;1259
                                    <!-- HIGHWIRE ID="149:1:4:2" -->
                                    <br>
                                    <A NAME="BIB3">
                                      <!-- null -->
                                    </A>
                                    <B>Chopra S, Athma P, Li XG, Peterson T
                                    </B> (1998) A maize Myb homolog is encoded by a multicopy gene complex. Mol Gen Genet 
                                    <B>260:
                                    </B> 372&#8211;380
                                    <!-- HIGHWIRE ID="149:1:4:3" -->
                                    <br>
                                    <A NAME="BIB4">
                                      <!-- null -->
                                    </A>
                                    <B>Davuluri RV, Sun H, Palaniswamy SK, Matthews N, Molina C, Kurtz M, Grotewold E
                                    </B> (2003) AGRIS: Arabidopsis gene regulatory information server, an information resource of Arabidopsis cis-regulatory elements and transcription factors. BMC Bioinformatics 
                                    <B>4:
                                    </B> 25
                                    <!-- HIGHWIRE ID="149:1:4:4" -->
                                    <br>
                                    <A NAME="BIB5">
                                      <!-- null -->
                                    </A>
                                    <B>Fornale S, Sonbol FM, Maes T, Capellades M, Puigdomenech P, Rigau J, Caparros-Ruiz D
                                    </B> (2006) Down-regulation of the maize and Arabidopsis thaliana caffeic acid O-methyl-transferase genes by two new maize R2R3-MYB transcription factors. Plant Mol Biol 
                                    <B>62:
                                    </B> 809&#8211;823
                                    <!-- HIGHWIRE ID="149:1:4:5" -->
                                    <br>
                                    <A NAME="BIB6">
                                      <!-- null -->
                                    </A>
                                    <B>Franken P, Schrell S, Peterson PA, Saedler H, Wienand U
                                    </B> (1994) Molecular analysis of protein domain function encoded by the 
                                    <I>myb
                                    </I>-homologous maize genes 
                                    <I>C1
                                    </I>, 
                                    <I>Zm 1
                                    </I> and 
                                    <I>Zm 38
                                    </I>. Plant J 
                                    <B>6:
                                    </B> 21&#8211;30
                                    <!-- HIGHWIRE ID="149:1:4:6" -->
                                    <br>
                                    <A NAME="BIB7">
                                      <!-- null -->
                                    </A>
                                    <B>Grotewold E
                                    </B> (2008) Transcription factors for predictive plant metabolic engineering: Are we there yet? Curr Opin Biotechnol 
                                    <B>19:
                                    </B> 138&#8211;144
                                    <!-- HIGHWIRE ID="149:1:4:7" -->
                                    <br>
                                    <A NAME="BIB8">
                                      <!-- null -->
                                    </A>
                                    <B>Grotewold E, Athma P, Peterson T
                                    </B> (1991) Alternatively spliced products of the maize P gene encode proteins with homology to the DNA-binding domain of myb-like transcription factors. Proc Natl Acad Sci USA 
                                    <B>88:
                                    </B> 4587&#8211;4591
                                    <!-- HIGHWIRE ID="149:1:4:8" -->
                                    <br>
                                    <A NAME="BIB9">
                                      <!-- null -->
                                    </A>
                                    <B>Guo A, He K, Liu D, Bai S, Gu X, Wei L, Luo J
                                    </B> (2005) DATF: a database of Arabidopsis transcription factors. Bioinformatics 
                                    <B>21:
                                    </B> 2568&#8211;2569
                                    <!-- HIGHWIRE ID="149:1:4:9" -->
                                    <br>
                                    <A NAME="BIB10">
                                      <!-- null -->
                                    </A>
                                    <B>Jakoby M, Weisshaar B, Droge-Laser W, Vicente-Carbajosa J, Tiedemann J, Kroj T, Parcy F
                                    </B> (2002) bZIP transcription factors in Arabidopsis. Trends Plant Sci 
                                    <B>7:
                                    </B> 106&#8211;111
                                    <!-- HIGHWIRE ID="149:1:4:10" -->
                                    <br>
                                    <A NAME="BIB11">
                                      <!-- null -->
                                    </A>
                                    <B>Jiang C, Gu J, Chopra S, Gu X, Peterson T
                                    </B> (2004) Ordered origin of the typical two- and three-repeat Myb genes. Gene 
                                    <B>326:
                                    </B> 13&#8211;22
                                    <!-- HIGHWIRE ID="149:1:4:11" -->
                                    <br>
                                    <A NAME="BIB12">
                                      <!-- null -->
                                    </A>
                                    <B>Ohnishi T, Sugahara S, Yamada T, Kikuchi K, Yoshiba Y, Hirano HY, Tsutsumi N
                                    </B> (2005) OsNAC6, a member of the NAC gene family, is induced by various stresses in rice. Genes Genet Syst 
                                    <B>80:
                                    </B> 135&#8211;139
                                    <!-- HIGHWIRE ID="149:1:4:12" -->
                                    <br>
                                    <A NAME="BIB13">
                                      <!-- null -->
                                    </A>
                                    <B>Rabinowicz PD, Braun EL, Wolfe AD, Bowen B, Grotewold E
                                    </B> (1999) Maize 
                                    <I>R2R3 Myb
                                    </I> genes: Sequence analysis reveals amplification in higher plants. Genetics 
                                    <B>153:
                                    </B> 427&#8211;444
                                    <!-- HIGHWIRE ID="149:1:4:13" -->
                                    <br>
                                    <A NAME="BIB14">
                                      <!-- null -->
                                    </A>
                                    <B>Stracke R, Werber M, Weisshaar B
                                    </B> (2001) The R2R3 MYB gene family in 
                                    <I>Arabidopsis thaliana
                                    </I>. Curr Opin Plant Biol 
                                    <B>4:
                                    </B> 447&#8211;456
                                    <!-- HIGHWIRE ID="149:1:4:14" -->
                                    <br>
                                    <A NAME="BIB15">
                                      <!-- null -->
                                    </A>
                                    <B>Wang BB, Brendel V
                                    </B> (2006) Genomewide comparative analysis of alternative splicing in plants. Proc Natl Acad Sci USA 
                                    <B>103:
                                    </B> 7175&#8211;7180
                                    <!-- HIGHWIRE ID="149:1:4:15" -->
                                    <br>
                                    <p>
                                    </p>
                                    <br><br>
<!--
                                    <hr>
                                    <A name="blast">
                                    </A>
                                    
                                    <h2>GRASSIUS BLAST sets
                                    </h2>
                                    Following sets are available for BLAST:
                                    <ol>
                                      <li>
                                        <b>All nt sequences:
                                        </b> Contains AGRIS, GRASSIUS and all Maize nucleotide sequences (#3 + #5 + #7)
                                      </li>
                                      <li>
                                        <b>ALL pep sequences:
                                        </b> Contains peptide sequences from AGRIS, GRASSIUS and Maize (#4 + #6 + #8)
                                      </li>
                                      <li>
                                        <b>GRASSIUS nt:
                                        </b> Contains nucleotide sequence of all transcription factors in GRASSIUS
                                      </li>
                                      <li>
                                        <b>GRASSIUS pep:
                                        </b> Peptide sequences of all transcription factors in GRASSIUS
                                      </li>
                                      <li>
                                        <b>AGRIS - TF nt:
                                        </b> Contains nucleotide sequence of all transcription factors in 
                                        <a href="http://arabidopsis.med.ohio-state.edu">AGRIS
                                        </a>
                                      </li>
                                      <li>
                                        <b>AGRIS - TF pep:
                                        </b> Contains peptide sequence of all transcription factors in 
                                        <a href="http://arabidopsis.med.ohio-state.edu">AGRIS
                                        </a>
                                      </li>
                                      <br> All Maize sequence information was obtained from maizesequence.org. Data is as of 03-21-2008. 
                                      </a>.
                                      <li>
                                        <b>Maize genes nt:
                                        </b> Contains nucleotide sequences of genes predicted as protein-coding, evidence, hypothetical and transposon-like
                                      </li>
                                      <li>
                                        <b>Maize gene translations:
                                        </b> Protein translations of genes that are classified as protein-coding, evidence, hypothetical and transposon-like
                                      </li>
                                      <li>
                                        <b>Maize BACs:
                                        </b> Raw maize genome sequences for accessioned BACs, as stored in GenBank
                                      </li>
                                      <li>
                                        <b>Maize BACs masked:
                                        </b> RepeatMasked maize genome sequences for accessioned BACs
                                      </li>
                                      <li>
                                        <b>Maize BAC contigs:
                                        </b> Sequences for all individual contigs that make up the accessioned BACs
                                      </li>
                                      <li>
                                        <b>Maize BAC contig masked:
                                        </b> RepeatMasked sequences for individual BAC-contigs
                                      </li>
                                    </ol>
                                    </p>
-->
                                    

<script>
    var $ = jQuery.noConflict();
    $(document).ready(function(){
        $('#nav_about').addClass("active")
    })
</script>

<?= $this->endSection() ?>